src/lib/getKcContext/KcContextBase.ts

```diff
@@ -215,6 +216,11 @@ export declare namespace KcContextBase {
         idpAlias: string;
     };
 
+    export type LoginIdpLinkEmail = Common & {
+        pageId: "login-idp-link-email.ftl";
+        idpAlias: string;
+    };
```

Comment:
```suggestion
    export type LoginIdpLinkEmail = Common & {
        pageId: "login-idp-link-email.ftl";
        idpAlias: string;
        brokerContext: {
            username: string;
        };
    };
```

Reply:
https://github.com/InseeFrLab/keycloakify/blob/main/src/lib/getKcContext/kcContextMocks/kcContextMocks.ts

Should be updated as well.  
You should add mock data for the new page.  🙏🏻